_includes/markdown/Vue.md

```diff
@@ -0,0 +1,110 @@
+```
+methods: {
+    addItem: function( event ) {
+        console.log( event );
+    }
+}
+```
+
+### Shorthand Syntax
+
+[Shorthand properties](https://vuejs.org/v2/guide/syntax.html#Shorthands) exist in Vue, to tighten up the HTML display when it can feel unnecessary to specifically call out each item as a Vue interaction. The framework specifically provides shorthand options for two of the most popular directives: `v-on` (events), and `v-bind` (attribute bindings).
+
+This allows you to specify events through the use of `@`. That replaces v-on: and would look like this `<button @click="submit">`. 
+
+If you are using Vue within a larger context you should try an avoid using shorthand properties as they can easily get lost in the HTML. However, if your project is a single page application and heavily using Vue, it may feel redundant to continuously preface these directives with `v-`, in that case you should use shorthand. You can [read more about shorthand properties](https://vuejs.org/v2/guide/syntax.html#Shorthands) in the official Vue documentation.
+
+## Templating and Components
+There are three main styles of templating built into the Vue framework:
+
+* Using a script block or inline template in the HTML (similar to Mustache or Handlebars)
+* JavaScript inline template strings (Similar to React)
+* Using an external `.vue` template file
+
+Of the three major options, which one you utilize will heavily depend on the type of project you’re building.
+
+If you are building small components that consume API data and do not need information from a WordPress template you can use your best judgement on whether the template references should remain in the HTML template or in the component JavaScript file. This is called an, “[inline template string](https://vuejs.org/v2/guide/components.html#Passing-Data-with-Props),” and you can easily pass data into the template through [props](https://vuejs.org/v2/guide/components.html#Props).
+
+If you are integrating your templating with WordPress templates and back end engineering, we recommend using a script block that is embedded into the HTML template so back and front end engineers can use the output as a point of reference and easily work on the project within their discipline. [Read more about inline templates](https://vuejs.org/v2/guide/components.html#Inline-Templates).
+
+However, if you are building a single page application with Vue, it is recommended that each project module be componentize within a `.vue` template file. All JavaScript, CSS, and HTML templates would be scoped together.  For this, you will also likely use the [Vue CLI](https://vuejs.org/v2/guide/installation.html#CLI) to initialize your project. These are called “[Single File Components](https://vuejs.org/v2/guide/single-file-components.html).”
+
+If you are using Vue outside of the SPA context make sure all your features have non-JS fallback behaviors so nothing appears broken if JavaScript fails to load.
```

Comment:
Are we still accounting for when a browser doesn't have JS turned on?

Reply:
It's case by case, but overall, yes. Especially in this context.